alonzo/impl/src/Cardano/Ledger/Alonzo/TxBody.hs

```diff
@@ -451,8 +455,8 @@ encodeTxBodyRaw
       ( \i ifee o f t c w u b rsh mi sh ah ni ->
           TxBodyRaw i ifee o c w f (ValidityInterval b t) u rsh mi sh ah ni
       )
-      !> Key 0 (E encodeFoldable _inputs)
-      !> Key 13 (E encodeFoldable _inputs_fee)
+      !> Key 13 (E encodeFoldable _inputs)
+      !> Key 0 (E encodeFoldable _inputs_fee)
       !> Key 1 (E encodeFoldable _outputs)
       !> Key 2 (To _txfee)
       !> encodeKeyedStrictMaybe 3 top
```

Comment:
Lets leave some comments here, about why we have the strange order of keys. Something to do with backwards comapatibility, and the fact all inputs in earlier Eras have to be considered for paying fees.